MergeContext.js

```diff
@@ -513,7 +513,7 @@ class MergeContext {
     }
 
     _prMessageValid() {
-        const lines = this._prMessage().split('\n');
+        const lines = this._prMessage().split(/\r*\n/);
```

Comment:
LGTM.

For the record, I tried to handle more encoding cases using a `$` regex specifically designed for end-of-line detection, but `$` does not match the preceding newline sequence itself (e.g., CR or CRLF), matches after CR in CRLF, and overall creates more problems (in this context) than it solves:

```
js> console.log("foo\r\nbar\rbaz\nbog".split(/$/m));
[ 'foo', '\r', '\nbar', '\rbaz', '\nbog' ]
```